MainModule/Client/UI/Default/FPS.lua

```diff
@@ -0,0 +1,43 @@
+client, service = nil, nil
+
+return function(data, env)
+	if env then
+		setfenv(1, env)
```

Comment:
dude this is cringe just directly use the passed env arguments and then define the services locally